app/models/presenters/standard_presenter.rb

```diff
@@ -8,8 +8,31 @@ module Presenters
   #
   class StandardPresenter < PlatePresenter
     include Presenters::Statemachine::Standard
+    include Presenters::Statemachine::FeatureInStates
 
     validates_with Validators::SuboptimalValidator
     validates_with Validators::ActiveRequestValidator
+
+    # Returns the CSV file links for the plate based on the configured states.
+    #
+    # @return [Array<Array<String, Array>>] the CSV file links
+    def csv_file_links
+      links =
+        purpose_config
+          .fetch(:file_links, [])
+          .select { |link| can_be_enabled?(link&.states) }
+          .map do |link|
+            [
+              link.name,
+              [
+                :limber_plate,
+                :export,
+                { id: link.id, limber_plate_id: human_barcode, format: :csv, **link.params || {} }
+              ]
+            ]
+          end
+      links << ['Download Worksheet CSV', { format: :csv }] if csv.present?
+      links
+    end
```

Comment:
Did you consider putting it in the PlatePresenter? There are lots of other Presenters that inherit from PlatePresenter that could potentially benefit from this.

Reply:
I have put into places where
- a state machine provided state is available
- file_links are customised

tube, plate and tube rack presenters where the concern is included do those two.

Reply:
I think, a better way of doing this is including the FeatureInStates concern in the top level presenter actually. I will add one liner for the file_links customisations.

Reply:
I have included the FeatureInStates concern in Presenter, removed csv_file_links override from StandardPresenter and updated csv_file_links of PlatePresenter with the can_be_enabled?(states) call on the links.